FLT/DivisionAlgebra/Finiteness.lean

```diff
@@ -169,8 +169,50 @@ lemma antidiag_mem_C {β : D_𝔸ˣ} (hβ : β ∈ ringHaarChar_ker D_𝔸) :
 
 end Aux
 
+def incl₂ : ringHaarChar_ker D_𝔸 → Prod D_𝔸 D_𝔸 :=
+  fun i => (i.1, i⁻¹.1)
+
+def M : Set (ringHaarChar_ker D_𝔸) := Set.preimage (incl₂ K D) (Aux.C K D)
+
+def MtoQuot : (ringHaarChar_ker D_𝔸) → (ringHaarChar_ker D_𝔸 ⧸
+    (MonoidHom.range (incl K D)).comap (ringHaarChar_ker D_𝔸).subtype) :=
+  fun a => Quot.mk _ a
```

Comment:
This already has a name, namely Quot.mk . You might want to avoid making too many definitions, they clog the place up. Have you considered `abbrev` instead? Quot.mk will have a ton of API and if you make MtoQuot as a def then it's suddenly much harder to use any of it because Lean won't unfold the definition. If you make it an abbrev then Lean just prints it as MtoQuot but in the actual term it's still Quot.mk so things like simp will work much better.